package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tourstream/fti-portal-styleguide",
-  "version": "0.20.1",
+  "version": "0.21.1",
```

Comment:
We don't change this version manually just leave it like it is and `npm version` will magically change this :)

Reply:
ah... right. forgot about that